frontend/views/containers/global-dashboard/NewsAndUpdates.vue

```diff
@@ -1,6 +1,8 @@
 <template lang='pug'>
 .c-news-and-updates-container
-  .c-post-block(v-for='(post, index) in dummyPosts' :key='index')
+  .c-loading(v-if='isLoading') {{ L('Loading news...') }}
+  .c-error(v-else-if='errorMessage') {{ L('Failed to load news: {errorMessage}', { errorMessage }) }}
+  .c-post-block(v-else v-for='(post, index) in posts' :key='index')
```

Comment:
Note that this `v-else v-for` thing seems like a mistake according to @SebinSong... and I'm inclined to agree as I've never seen this, OTOH it worked just fine?